### PR TITLE
Improve UI

### PR DESCRIPTION
**Pull request by A7abhilash**
## What is the change?
Change in home UI

## Related issue?
#333  
 Old PR: #339  #357 

## How was it tested?
Tested thoroughly in localhost

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
Add a screenshot or demo video if appropriate.

### CODE REVIEW

**Comment by Pranav016** [comment]
Can you add some demo video of what UI improvements were made ?

Also, some code I saw wasn't indented properly so I would recommend you to do that since we want to maintain a good code base.

**Comment by A7abhilash** [comment]
@Pranav016 
The size of screen recording was more than 10mb. I tried compressing it online but didnt work and it showed failed to comprees, idk why so!!

**Comment by Pranav016** [comment]
> @Pranav016
> The size of screen recording was more than 10mb. I tried compressing it online but didnt work and it showed failed to comprees, idk why so!!

Turn the video to gif

**Comment by Pranav016** [comment]
> @Pranav016
> The size of screen recording was more than 10mb. I tried compressing it online but didnt work and it showed failed to comprees, idk why so!!
> 
> 

Are these responsive ?

**Comment by A7abhilash** [comment]
> > @Pranav016
> > The size of screen recording was more than 10mb. I tried compressing it online but didnt work and it showed failed to comprees, idk why so!!
> > 
> > 
> 
> Are these responsive ?

100% yes.... I have used bootstrap column system for making them responsive

**Comment by Pranav016** [comment]
> Can you add some demo video of what UI improvements were made ?
> 
> Also, some code I saw wasn't indented properly so I would recommend you to do that since we want to maintain a good code base.

Ok, great! Can you indent the code and resolve conflicts?

**Comment by Pranav016** [comment]
> 100% yes.... I have used bootstrap column system for making them responsive

Ok, great! Can you indent the code and resolve conflicts?

File: views/home.ejs [hunk 1]
```diff
@@ -1,63 +1,31 @@
 <%- include("partials/header", {isAuthenticated}) -%>
-<link rel="stylesheet" href="/css/styles.css">
-<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"></link>
```
Comment (Pranav016) [inline views/home.ejs:3]: these were removed ?

File: views/home.ejs [hunk 2]
```diff
@@ -1,63 +1,31 @@
...
-        <form action="/search" method="POST">
-            <div class="form-group search-form">
-                <input class="form-control" type="text" name="query" placeholder="Type down your query"autocomplete="off"  value="<%= search %>" required></input>
-                <button class="button  search" style="margin-left: 1em;">Search</button>
-            </div>
-        </form>
-    </div>
-
-    <!-- display the search results -->
-    <!-- if the page was rendered by querying in searchbar -->
```
Comment (Pranav016) [inline views/home.ejs:25]: Comments

**Review by Pranav016 — APPROVED** [review]
Merging this PR for now, some comments were removed that were there to label different sections of code, it would be great if you could make a follow up PR to add those :)

**Comment by Pranav016** [comment]
/gssoc/

**Comment by Pranav016** [comment]
/level2/

**Pull request merged**